### PR TITLE
chore: re-enable integration tests

### PR DESCRIPTION
**Pull request by toddbaert**
Update test harness and re-enable integration test profile

closes: https://github.com/open-feature/java-sdk/issues/113

### CODE REVIEW

**Review by beeme1mr — APPROVED** [review]

**Comment by toddbaert** [comment]
In most cases I would wait for approval from @justinabrahms but this is only re-enabling existing testing.

**Pull request merged**
